libs/src/utils/utils.test.ts

```diff
@@ -1,21 +1,21 @@
-const assert = require('assert')
-const utils = require('./utils')
+import assert from 'assert'
+import Utils from './utils'
```

Comment:
same q here about destructuring in import

Reply:
same answer, going to update though